extensions/cornerstone/src/services/ViewportService/CornerstoneViewportService.ts

```diff
@@ -39,6 +39,8 @@ class CornerstoneViewportService implements IViewportService {
   renderingEngine: Types.IRenderingEngine | null;
   viewportsInfo: Map<number, ViewportInfo>;
   viewportGridResizeObserver: ResizeObserver | null;
+  // TODO - get the right type here.
+  hangingProtocolService: object;
```

Comment:
I'm not sure if we should use `any` or `object`, I haven't seen object anywhere. @ladeirarodolfo

Reply:
They are fairly different - any includes  number, string, function, whereas object means a JavaScript object value, so I'd prefer to be more descriptive here since we actually know that it is an object and NOT just any old value.